### PR TITLE
docs: Update license to 2021

### PR DESCRIPTION
**Pull request by haworku**
# Summary
It's a new year. Also there were a bunch of broken links 🤕 from an earlier refactor.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -174,28 +174,3 @@
...
-      To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
-      replaced with your own identifying information. (Don't include
-      the brackets!)  The text should be enclosed in the appropriate
-      comment syntax for the file format. We also recommend that a
-      file or class name and description of purpose be included on the
-      same "printed page" as the copyright notice for easier
-      identification within third-party archives.
-
-   Copyright 2019, TrussWorks, Inc.
```
Comment (ahobson) [inline LICENSE:189]: I don't think we need that APPENDIX paragraph, but don't we want the bits below, including the Copyright?

**Review by ahobson — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (haworku) [reply LICENSE:189]: Hmmm... so this text is what we already have main README - See https://github.com/trussworks/react-uswds#license 

By keeping it here too we have to update things in 2 places.  Pretty sure we have things proper in the README alone is enough. And then this is just a link to actual text.  

Thoughts?

Comment (ahobson) [reply LICENSE:189]: Ok, I see now. Thank you for explaining!

**Review by ahobson — APPROVED** [review]
👍

**Review by brandonlenz — APPROVED** [review]

**Pull request merged**